frappe/public/scss/page-builder.scss

```diff
@@ -327,7 +327,7 @@
 .testimonial-logo img {
 	display: inline-block;
 	max-width: 10rem;
-	max-height: 2.5rem;
+	max-height: 10rem;
```

Comment:
??

Reply:
![image](https://user-images.githubusercontent.com/42651287/90524832-2b848e80-e18c-11ea-841d-7f73b30a07f1.png)

For showing user images in testimonials